src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -330,6 +342,12 @@ class CallOpcodesSpec extends WordSpec with Matchers with PropertyChecks {
         call.stateOut.gasRefund shouldBe 0
       }
 
+      "destruct ether if own address equals refund address" in {
```

Comment:
It's a good test :+1:
But the primary place for unit tests for opcodes other than *CALL* and CREATE is `OpCodeFunSpec`. Could we add a test case there? (Note these are property tests with generators for general cases, and tables for specific ones)